Documentation~/RandomTerrainTile.md

```diff
@@ -0,0 +1,33 @@
+# Random Terrain Tile
+
```

Comment:
Thanks for writing the documentation for this!

A small addition would be adding a contribution note with your username too, for example in https://github.com/Unity-Technologies/2d-extras/blob/master/Documentation~/LineBrush.md . It would be good acknowledgement of your work here!